theme/views/@layout.latte

```diff
@@ -19,14 +19,14 @@
 
 	{* {use_google_fonts([ 'Lato:400,700,900:latin,latin-ext', 'Playfair+Display:700italic:latin,latin-ext' ])} *}
 
-	<link rel="stylesheet" href="{$assetsPath}/styles/index.css" media="screen">
-	<link rel="stylesheet" href="{$assetsPath}/styles/print.css" media="print">
+	<link rel="stylesheet" href="{$assetsPath}/styles/{$buildstamp}index.css" media="screen">
+	<link rel="stylesheet" href="{$assetsPath}/styles/{$buildstamp}print.css" media="print">
 
 	<link rel="icon" type="image/png" href="{$assetsPath}/favicon.png" sizes="16x16">
 	<link rel="icon" type="image/png" href="{$assetsPath}/favicon-32x32.png" sizes="32x32">
 
 	<!--[if lt IE 9]>
-	<link rel="stylesheet" href="{$assetsPath}/styles/oldie.css" media="screen">
+	<link rel="stylesheet" href="{$assetsPath}/styles/{$buildstamp|noescape}oldie.css" media="screen">
```

Comment:
U ostatních to funguje bez toho. Akorát tady se context-aware escaping asi nějak snažilo brát v potaz `<!--[if lt IE 9]>`.